homework1/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace Homework1
+{
+    /// <summary>
+    /// Represents a bank account with a balance.
+    /// </summary>
+    class BankAccount
+    {
+        private int _balance;
+
+        /// <summary>
+        /// Initializes a new instance of the BankAccount class with a zero balance.
+        /// </summary>
+        public BankAccount()
+        {
+            _balance = 0;
+        }
+
+        /// <summary>
+        /// Deposits the specified amount into the account.
+        /// </summary>
+        /// <param name="amount">The amount to deposit.</param>
+        public void Deposit(int amount)
+        {
+            _balance += amount;
+        }
+
+        /// <summary>
+        /// Withdraws the specified amount from the account if sufficient balance is available.
+        /// </summary>
+        /// <param name="amount">The amount to withdraw.</param>
+        public void Withdraw(int amount)

```

Comment:
You can use the ternary operator.
Something like -
_balance = amount <= _balance ? _balance - amount : _balance;